bioconda_utils/cli.py

```diff
@@ -20,11 +21,49 @@
 from . import github_integration
 from . import bioconductor_skeleton as _bioconductor_skeleton
 from . import cran_skeleton
-from .update_pinnings import should_be_bumped, bump_recipe
+from . import update_pinnings
+from . import graph
 
 logger = logging.getLogger(__name__)
 
 
+def enable_logging(default_loglevel='info'):
+    """Decorator setting up logging for argh command
+
+    - Adds the parameter ``--loglevel`` with a default of ``info``
+      and sets up the logging handlers appropriately.
+    - Adds the paremeter ``--pdb`` (or ``-P``) to enable dropping
+      into PDB if an unhandled exception is encountered.
+
+    >>> @enable_logging()
+    >>> def command():
+    >>>   pass
+
+    >>> @enable_logging('warn')
+    >>> def command():
+    >>>   pass
+
+    Args:
+      default_loglevel: loglevel used when --loglevel is not passed
+    """
+    def decorator(func):
+        @arg('--loglevel', help="Set logging level (debug, info, warning, error, critical)")
+        @arg('-P', '--pdb', help="Drop into debugger on exception")
```

Comment:
Probably should be added via it's own decorator, but I really wanted the "if you crash, drop into PDB" feature.